pages/components/alert/alert.css

```diff
@@ -0,0 +1,87 @@
+/* Importing external styles */
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.alert-wrapper {
+  width: 40vw;
+  border: 0.5px solid lightgray;
+  border-radius: 5px;
+}
+
+.alert {
+  border: none;
+  padding: 1rem;
+  margin: 1rem 0;
+  border-radius: 10px;
+  width: max-content;
+}
+
+/* Standard Alert */
+
+.alert-primary {
+  background-color: var(--primary-color);
+  color: #fff;
```

Comment:
try to add `#fff` inside variables to make everything generic

Reply:
Added the changes.